src/app/components/navbar/navbar.module.css

```diff
@@ -87,6 +87,7 @@
 @media (max-width: 1150px) {
   .home {
     display: none;
+    font-weight: 900;
```

Comment:
This is not in the correct location. This is adding the font-weight to the name once we hide it in mobile view. the display:none hides the text.